data/translations/ja.ts

```diff
@@ -69,5 +69,49 @@
         <source>Select your user and enter password</source>
         <translation>ユーザ名を選び、パスワードを入力してください</translation>
     </message>
+    <message>
+        <source>Please enter a password!</source>
+        <translation>パスワードを入力してください!</translation>
```

Comment:
In the Japanese context, adding an exclamation mark (!) here sounds like a bit too strong.
To avoid letting users feel offended or blamed, I suggest removing the exclamation mark.

```suggestion
        <translation>パスワードを入力してください</translation>
```